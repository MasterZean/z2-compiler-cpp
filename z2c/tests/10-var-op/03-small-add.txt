{ val a = Small{} + Bool{} }
{ val a = Small{} + Small{} }
{ val a = Small{} + Byte{} }
{ val a = Small{} + Short{} }
{ val a = Small{} + Word{} }
{ val a = Small{} + Int{} }
{ val a = Small{} + DWord{} }
{ val a = Small{} + Long{} }
{ val a = Small{} + QWord{} }
{ val a = Small{} + Float{} }
{ val a = Small{} + Double{} }
{ val a = Small{} + Char{} }
{ val a = Small{} + PtrSize{} }
