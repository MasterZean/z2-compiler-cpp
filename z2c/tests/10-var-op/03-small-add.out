{
	int8 a = 0 + false;
}
{
	int8 a = 0 + 0;
}
{
	uint8 a = 0 + 0u;
}
{
	int16 a = 0 + 0;
}
{
	uint16 a = 0 + 0u;
}
{
	int32 a = 0 + 0;
}
{
	uint32 a = 0 + 0u;
}
{
	int64 a = 0 + 0ll;
}
{
	uint64 a = 0 + 0ull;
}
{
	float a = 0 + 0.0f;
}
{
	double a = 0 + 0.0;
}
{
	uint32 a = 0 + '\0';
}
{
	size_t a = 0 + 0u;
}
